pkg/consts/qos.go

```diff
@@ -61,3 +61,18 @@ const (
 	PodAnnotationNetworkEnhancementAffinityRestricted     = "topology_affinity_restricted"
 	PodAnnotationNetworkEnhancementAffinityRestrictedTrue = "true"
 )
+
+// const variables for pod annotations about qos level enhancement for micro-topology level inter-pod affinity
```

Comment:
please more detailed introduction for those enhancements

Reply:
OK, more introduction has been added.